frontend/next.config.js

```diff
@@ -146,7 +146,38 @@ module.exports = {
       ),
     },
   },
-  webpack: (config, { webpack }) => {
+
+  webpack: (config, { webpack, dev }) => {
+    if (dev) {
+      config.module.rules.push({
+        // This rule is to enable SVG imports when experimental-https is set.
+        // experimental-https prevents SVGs from being imported correctly on local servers
+        test: /\.svg$/,
+        loader: "next-image-loader",
+        issuer: {
+          and: [/\.(ts|tsx|js|jsx|md|mdx)$/],
+        },
+        options: {
+          isDev: true,
+          compilerType: "server",
+          basePath: "",
+          assetPrefix: "",
+        },
+        exclude: path.resolve(__dirname, "src/common/icons"),
+      });
+    }
+
+    config.module.rules.push({
```

Comment:
This will close this 3 year old ticket 😆 👏 Thanks so much for doing this!

#1553